packages/core/admin/server/tests/admin-api-token-crud.test.api.js

```diff
@@ -22,13 +24,17 @@ describe('Admin API Token v2 CRUD (api)', () => {
   beforeAll(async () => {
     strapi = await createStrapiInstance();
     rq = await createAuthRequest({ strapi });
+    // To eliminate latency in the request and predict the expiry timestamp, we freeze Date.now()
+    now = Date.now();
+    nowSpy = jest.spyOn(Date, 'now').mockImplementation(() => now);
```

Comment:
mysql2 had issues with `jest.useFakeTimers().setSystemTime()`, but this is actually an improvement on that because mocking Date.now() is much more precise / less extreme than faking the system time.